LPFilter/Source/PluginProcessor.h

```diff
@@ -61,13 +61,20 @@ class LpfilterAudioProcessor  : public AudioProcessor
 
 private:
     
-    void process (dsp::ProcessContextReplacing<float>) noexcept;
+    void process (AudioSampleBuffer& buffer) noexcept;
     //==============================================================================
     AudioParameterFloat* gain;
     AudioParameterFloat* frequency;
+
+    Dsp::Params paramsDsp;
     float defaultFreq = 60.f;
     
     dsp::ProcessorDuplicator<dsp::IIR::Filter<float>, dsp::IIR::Coefficients<float>> lpfJuce;
     
+    ScopedPointer<Dsp::Filter> lpfDspLib;
+    
+    AudioSampleBuffer tempBuffer;
```

Comment:
This `tempBuffer` is killing me! Please find a better name. For example, `processedBuffer`or `filterBuffer` etc